src/components/FlightsTable/FlightsTable.jsx

```diff
@@ -7,9 +7,9 @@ export const FlightsTable = ({originFilter, flights}) => {
     <section className="flightsTable">
       <h3>Flights</h3>
       <section>
```

Comment:
Section inside a section? And then `FlightsTableRow` is another section? Not cool!

Since this is a list of flights (not a table, apparently, haha) you'll probably want a `ul`:

```
<ul>
    {flights.map( ..... => <FlightsTableRow ......> )}
</ul>
```

with `FlightsTableRow` being wrapped in `<li>`. Then you'll need to set `list-style: none` styling on the `ul` and undo some of the default margins and padding but that would be more semantically correct.